src/components/modals/DeleteModal.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import ReactModal from 'react-modal';
+import { useModal } from 'react-modal-hook';
+import styled from 'styled-components';
+import {
+  modalStyles,
+  PrimaryButtonFilled,
+  CloseIcon,
+  ModalActions,
+} from '../../shared/modalstyles';
+
+const deleteModalStyles = {
+  content: {
+    maxWidth: '670px',
+    maxHeight: '350px',
+    margin: '0 auto',
+  },
+};
+
+const ModalContent = styled.div`
+  margin-top: 50px;
+  margin-bottom: 90px;
+`;
+
+const useDeleteMovieModal = () => {
```

Comment:
Almost all comments reported for 'AddEditModal' apply to this component as well.